policies/ucs_server_policies/misc/ssh_policy/terraform/ssh_policy.tf

```diff
@@ -1,3 +1,22 @@
+provider "intersight" {
+  endpoint        = "https://intersight.com"
+  apikey          = "xxxxx27564612d30dxxxxx/5f21c9d97564612d30dd575a/5f9a8b877564612xxxxxxxx"
+  secretkey       = "C:\\secretKey.txt"
+}
+
+resource "intersight_server_profile" "server1" {
```

Comment:
remove this server profile creation and reference in ssh policy

Reply:
Removed and referenced.